tests/lax_numpy_test.py

```diff
@@ -1252,6 +1252,40 @@ def testCumSumProd(self, axis, shape, dtype, out_dtype, onp_op, jnp_op, rng_fact
                             tol=tol)
     self._CompileAndCheck(jnp_fun, args_maker, check_dtypes=True)
 
+
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "_yshape={}_xshape={}_dx={}_axis={}".format(
+        jtu.format_shape_dtype_string(yshape, dtype),
+        jtu.format_shape_dtype_string(xshape, dtype) if xshape is not None else None,
+        dx,
+        axis),
+        "yshape": yshape,
```

Comment:
We would usually pack tighter vertically.

Reply:
Done.